src/core/Entry.cpp

```diff
@@ -325,25 +325,28 @@ QString Entry::totp() const{
 
         int rc = oath_init();
```

Comment:
You are wrong. Q_UNUSED should only wrap unused parameters and not return values.